Decred Wallet/Features/TransactionDetails/TransactionDetails.storyboard

```diff
@@ -18,67 +21,430 @@
+                                                                <constraint firstItem="cuK-2M-WpJ" firstAttribute="leading" secondItem="uqH-st-ioR" secondAttribute="leading" constant="16" id="fug-4p-MuO"/>
+                                                                <constraint firstAttribute="trailing" secondItem="cuK-2M-WpJ" secondAttribute="trailing" constant="16" id="glf-25-k9t"/>
+                                                            </constraints>
+                                                        </tableViewCellContentView>
+                                                        <connections>
+                                                            <outlet property="titleLabel" destination="cpj-bh-TIm" id="T2F-87-anA"/>
+                                                            <outlet property="valueButton" destination="6PO-NB-DP1" id="Dfx-HE-2RI"/>
+                                                        </connections>
+                                                    </tableViewCell>
+                                                    <tableViewCell clipsSubviews="YES" contentMode="scaleToFill" preservesSuperviewLayoutMargins="YES" selectionStyle="none" indentationWidth="10" reuseIdentifier="TransactionInputDetailCell" rowHeight="78" id="HN0-ox-bOO" customClass="TransactionInputDetailCell" customModule="Decred_Wallet" customModuleProvider="target">
+                                                        <rect key="frame" x="0.0" y="176" width="375" height="78"/>
+                                                        <autoresizingMask key="autoresizingMask"/>
+                                                        <tableViewCellContentView key="contentView" opaque="NO" clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="center" preservesSuperviewLayoutMargins="YES" insetsLayoutMarginsFromSafeArea="NO" tableViewCell="HN0-ox-bOO" id="gbE-qe-SCJ">
+                                                            <rect key="frame" x="0.0" y="0.0" width="375" height="78"/>
+                                                            <autoresizingMask key="autoresizingMask"/>
+                                                            <subviews>
+                                                                <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="8ge-Wq-XEj" customClass="RoundedView" customModule="Decred_Wallet" customModuleProvider="target">
+                                                                    <rect key="frame" x="16" y="0.0" width="343" height="70"/>
+                                                                    <subviews>
+                                                                        <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="Iea-G3-S8B">
```

Comment:
This `<view/>` is the only subview of the parent `RoundView`. The subviews of this view could have been direct subviews of the parent `RoundView` eliminating the need for an intermediary view.

Both `TransactionInputDetailCell` and `TransactionOutputDetailCell` are affected.